core/sms_server.py

```
@@ -0,0 +1,1422 @@
+    logger.info("Starting advanced batch processor...")
+    
+    while True:
+        try:
+            pool = await get_db_pool()
+            
+            # Read batch size and timeout settings
+            try:
+                batch_size = int(await get_setting('batch_size'))
+                try:
+                    batch_timeout = float(await get_setting('batch_timeout'))
+                except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)